keps/sig-scheduling/1819-scheduler-extender/README.md

```diff
@@ -185,6 +185,13 @@ extenders:
   ignorable: false
 ```
 
+Multiple extenders can be configured and will be called sequentially in the
+scheduler framework. Like the scheduler framework running filter plugins, which
+will stop on the first filter that returns `Unschedulable` status,
+unschedulable nodes will be not be passed to subsequent extenders. Users are
+encouraged to carefully order extenders to have the extender which may report
+`UnschedulableAndUnresolvable` first. This can improve preemption latency.
```

Comment:
This is a bit vague to me, I think we should separate the explanation of "extenders are running sequentially" and "semantics of failedAndUnresolvable" - the previous one was already known; while the latter one is new and should be emphasized. See https://github.com/kubernetes/kubernetes/pull/92866#discussion_r552245023.

Reply:
updated

I add a note about the order of multiple extenders in the "Configuration" section. The explanation of "semantics of failedAndUnresolvable" is moved to the "Interface > Filter" section, right after the spec of the `ExtenderFilterResult` struct.

What do you think?

Reply:
Sounds better. Some comments below.